mods/d2k/rules/defaults.yaml

```diff
@@ -336,8 +336,13 @@
 		VisibilityType: CenterPosition
 	Targetable:
 		TargetTypes: Ground, C4, Structure
+	BodyOrientation:
+		CameraPitch: 255
```

Comment:
The tradeoff is that this then breaks everything else that relies on having the correct-for-artwork/correct-for-mod camera pitch.  Finding and then scaling the y offsets of the targetable positions is far preferrable to that.